README.md

```diff
@@ -417,6 +417,10 @@ this behavior, we can use the `-I` (or `--no-ignore`) option:
 target/debug/deps/libnum_cpus-f5ce7ef99006aa05.rlib
 ```
 
+*fd* respects the global git-ignore file as well. But as said above: Only when the directory belongs to a git repository.
+
+*fd* reads and uses ignore-files (`.ignore` and `.fdignore`), for the directory containing the ignore-file and its sub-directories. But a global `~/.ignore-file` or `~/.fdignore` are not used.
+
```

Comment:
```suggestion
But global `~/.ignore` or `~/.fdignore` are not used outside of `~`.
```

Reply:
Really? I try to avoid 'negations' like that. Calling them "global" and then saying they are only used in "~" would confuse me. 

Would it be better to say:
```
.ignore and .fdignore files are only considered in the directory they are in (and the subdirectories). That means: .ignore and .fdignore in the HOME directory are not considered globally, but only for the HOME directory and its subdirectories
```